packages/suite/src/components/onboarding/UnexpectedState/components/IsSameDevice.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
-import { FormattedMessage } from 'react-intl';
+import { Translation } from '@suite-components/IntlMessageExtractor';
```

Comment:
Would not be better to call it `@suite-components/Dictionary` or `@suite-components/Translate`
instead of `@suite-components/IntlMessageExtractor`?